src/test/java/com/xceptance/neodymium/tests/NeodymiumTestSelfTest.java

```diff
@@ -0,0 +1,177 @@
+package com.xceptance.neodymium.tests;
+
+import java.lang.reflect.Field;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestName;
+import org.junit.runner.Description;
+import org.junit.runner.Result;
+import org.junit.runner.notification.Failure;
+
+public class NeodymiumTestSelfTest extends NeodymiumTest
+{
+    @Rule
+    public TestName name = new TestName();
+
+    @Test
+    public void testCheckFailedOneFromOne() throws NoSuchFieldException, SecurityException, IllegalArgumentException, IllegalAccessException
+    {
+        final String errorMessage = "This is RuntimeException 1";
+        Result result = createResult(1, 0, new HashMap<String, Throwable>()
```

Comment:
* Variable `result` should be `final`
* Building a map became quite simple since Java 9:  
  `Map.of(name.getMethodName().concat("1", new RuntimeException(errorMessage))`

Reply:
Done